LoopKitUI/Views/Settings Editors/InsulinModelSelection.swift

```diff
@@ -11,18 +11,19 @@ import SwiftUI
 import LoopKit
 
 public struct InsulinModelSelection: View {
+    @EnvironmentObject private var displayGlucoseUnitObservable: DisplayGlucoseUnitObservable
     @Environment(\.appName) private var appName   
-    @Environment(\.dismissAction) var dismiss
-    @Environment(\.authenticate) var authenticate
-
-    let initialValue: InsulinModelSettings
-    @State var value: InsulinModelSettings
-    let insulinSensitivitySchedule: InsulinSensitivitySchedule
-    let glucoseUnit: HKUnit
-    let supportedModelSettings: SupportedInsulinModelSettings
-    let mode: SettingsPresentationMode
-    let save: (_ insulinModelSettings: InsulinModelSettings) -> Void
-    let chartManager: ChartsManager
+    @Environment(\.dismissAction) private var dismiss
+    @Environment(\.authenticate) private var authenticate
+
+    @State private var value: InsulinModelSettings
+    @State private var chartManager: ChartsManager
+
+    private let initialValue: InsulinModelSettings
+    private let insulinSensitivitySchedule: InsulinSensitivitySchedule
+    private let supportedInsulinModelSettings: SupportedInsulinModelSettings
+    private let mode: SettingsPresentationMode
+    private let saveInsulinModelSelection: (_ insulinModelSettings: InsulinModelSettings) -> Void
```

Comment:
[nit] Why the change from `save` to `saveInsulinModelSelection`?

Reply:
No real reason, I just find it clearer. I'll revert.